.travis.yml

```
@@ -0,0 +1,53 @@
+### Project specific config ###
+
+language: generic
+
+matrix:
+  include:
+    - os: linux
+      env: ATOM_CHANNEL=stable
+
+    - os: linux
+      env: ATOM_CHANNEL=beta
+
+env:
+  global:
+    APM_TEST_PACKAGES: "language-lua"
+
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install --assume-yes --quiet --no-install-suggests --no-install-recommends lua5.1 luajit
```

```suggestion
  - sudo apt-get install --assume-yes --quiet --no-install-suggests --no-install-recommends lua5.3 luajit
```
Again, might as well use the current version.